packages/llvm6.rb

```diff
@@ -0,0 +1,313 @@
+      puts "Openmp source code exists and perform unpacking".lightred
+      system "tar", "xf", "openmp-6.0.0.src.tar.xz", "-C", "projects"
+      puts "Openmp source code unpacked".lightgreen
+      system "rm -rf projects/openmp"
+      system "mv -v projects/openmp-6.0.0.src projects/openmp"
+    else
+      puts "Downloading openmp".lightred
+      system('curl', '-s', '-C', '-', '--insecure', '-L', '-#', url_openmp, '-o', filename_openmp)
+      abort 'Checksum mismatch. :/ Try again.'.lightred unless
+        Digest::SHA256.hexdigest( File.read("./#{filename_openmp}") ) == sha256sum_openmp
+      puts "Openmp archive downloaded".lightgreen
+      
+      system "tar", "xf", "openmp-6.0.0.src.tar.xz", "-C", "projects"
+      puts "Openmp source code unpacked".lightgreen
+      system "mv -v projects/openmp-6.0.0.src projects/openmp"
+    end
+ 
+    ############################# Download libunwind (procjects) ####################################
+    url_libunwind = 'https://releases.llvm.org/6.0.0/libunwind-6.0.0.src.tar.xz'
+    uri_libunwind = URI.parse url_libunwind
+    filename_libunwind = File.basename(uri_libunwind.path)
+    sha256sum_libunwind = '256c4ed971191bde42208386c8d39e5143fa4afd098e03bd2c140c878c63f1d6'
+
+    if File.exist?(filename_libunwind) && Digest::SHA256.hexdigest( File.read("./#{filename_libunwind}") ) == sha256sum_libunwind
+      puts "Libunwind source code exists and perform unpacking".lightred
+      system "tar", "xf", "libunwind-6.0.0.src.tar.xz", "-C", "projects"
+      puts "Libunwind source code unpacked".lightgreen
+      system "rm -rf projects/libunwind"
+      system "mv -v projects/libunwind-6.0.0.src projects/libunwind"
+    else
+      puts "Downloading libunwind".lightred
+      system('curl', '-s', '-C', '-', '--insecure', '-L', '-#', url_libunwind, '-o', filename_libunwind)
+      abort 'Checksum mismatch. :/ Try again.'.lightred unless
+        Digest::SHA256.hexdigest( File.read("./#{filename_libunwind}") ) == sha256sum_libunwind
+      puts "Libunwind archive downloaded".lightgreen
+      
+      system "tar", "xf", "libunwind-6.0.0.src.tar.xz", "-C", "projects"
+      puts "Libunwind source code unpacked".lightgreen
+      system "mv -v projects/libunwind-6.0.0.src projects/libunwind"
+    end
+ 
+  
+     ############################# Download test-suite (procjects) ####################################
+    url_testsuite = 'https://releases.llvm.org/6.0.0/test-suite-6.0.0.src.tar.xz'
+    uri_testsuite = URI.parse url_testsuite
+    filename_testsuite = File.basename(uri_testsuite.path)
+    sha256sum_testsuite = '74e0055efa27b2143415148ee93b817155e6333337d9cadd4cc5d468ad3c0edf'
+
+    if File.exist?(filename_testsuite) && Digest::SHA256.hexdigest( File.read("./#{filename_testsuite}") ) == sha256sum_testsuite
+      puts "Test-suite source code exists and perform unpacking".lightred
```

Comment:
Yellow instead of red?

Reply:
Just keep it red. No one have interest to compile it from source code ... This kind of color is only helpful to develop a method to compile it.

Reply:
Right but red generally indicates an error.  In this case, it's just a status message so it should be a different color.